.circleci/config.yml

```diff
@@ -7,6 +7,11 @@ jobs:
   build:
     docker:
       - image: node:8
+      # See https://github.com/cypress-io/cypress-docker-images
+      - image: cypress/base:8
```

Comment:
@k-pammi you're using two images, probably you can get rid of `node:8` - to see working cypress tests on Circle check out config in https://github.com/comicrelief/grants-eligibility-form/blob/master/.circleci/config.yml